src/controllers/exchanger.controller.ts

```diff
@@ -0,0 +1,13 @@
+import { Request, Response } from 'express';
+import { ExchangerService } from '../services/exhanger.service';
+import { StatusCode } from '../models/status-codes.model';
+
+export class ExchangerController {
+  async getCurrentRate(req: Request, res: Response) {
+    const currentRate = await ExchangerService.getCurrentRate();
+    if (!currentRate) {
+      res.status(StatusCode.Invalid).json('Invalid status value');
+    }
+    res.status(StatusCode.Success).json(currentRate);
+  }
+}
```

Comment:
It is a good approach to export already initialized class if there is no constructor, so it won't be done each time being imported, but it's not a big deal.
e.g.
`export default new ExchangeController()`

Reply:
Thanks! I decided to add constructor to the SubscriptionController (as it depens on the service, and if it'll be extended, there is no need to create a new service instance each time)
And also I changed exchanger controller to be a single function, as there is no need of service instance currently